README.asciidoc

```diff
@@ -26,9 +26,150 @@ If you prefer to install and manage Elasticsearch yourself, you can download
 the latest version from 
 https://www.elastic.co/downloads/elasticsearch[elastic.co/downloads/elasticsearch].
 
-For more installation options, see the
-https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html[Elasticsearch installation
-documentation].
+=== Set up a local dev deployment
+
+To try out Elasticsearch on your dev machine, we recommend using Docker
+and running both Elasticsearch and Kibana.
+Docker images are available from the https://www.docker.elastic.co[Elastic Docker registry].
+
+For other installation options, see the
+https://www.elastic.co/guide/en/elasticsearch/reference/current/install-elasticsearch.html[Elasticsearch installation documentation].
+
+**Install and run Elasticsearch**
+
+. Install and start https://www.docker.com/products/docker-desktop[Docker
+Desktop]. Go to **Preferences > Resources > Advanced** and set Memory to at least 4GB.
+
+. Start an Elasticsearch container:
++
+----
+docker network create elastic
+docker pull docker.elastic.co/elasticsearch/elasticsearch:{version}
```

Comment:
I don't know that the `{version}` variable will render the current Elasticsearch version in the README. Instead, should we add a note informing users to replace `{version}` with the version that they want to download, such as `8.4.1`?

Same comment in the next command.

Reply:
It won't render. I do not know of a way to do substitution that will work when viewed on GH.  A callout is probably the best we can do.

Reply:
I think that a callout works well here. Something simple like:

>Replace `{version}` with the version of Elasticsearch that you want to install.

Totally at your discretion.